extra/systemd/foreman-proxy.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Foreman Proxy
+Wants=basic.target
+After=basic.target network.target
+
+[Service]
+Type=notify
+User=foreman-proxy
+ExecStart=/usr/share/foreman-proxy/bin/smart-proxy --no-daemonize
+EnvironmentFile=-/etc/sysconfig/foreman-proxy
```

Comment:
This is RH specific. Debian has ```/etc/default``` but I'm wondering if the environment is even used. If users need it they can override it in ```/etc/systemd/system/foreman-proxy.service.d/override.conf```.